doc/man1/openssl-list.pod.in

```diff
@@ -97,6 +104,10 @@ displayed.
 In verbose mode, the algorithms provided by a provider will get additional
 information on what parameters each implementation supports.
 
+=item B<-random-instances>
+
+List the primary, pubic and private random number generator details.
```

Comment:
Another typo, that - although mildly entertaining - should be fixed 🙃 
```suggestion
List the primary, public and private random number generator details.
```

Reply:
Oops, missed this one...

Reply:
Yeah :sweat_smile:
Good spotting - I had copied this text over from `apps/list.c`, where the typo has bee sitting since Sep 22.